swagger.yaml

```diff
@@ -295,6 +295,106 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/downloads/image.iso:
```

Comment:
I think that the naming is really confusing with the `/image` endpoint.
I couldn't guess that this downloads only the headers. Maybe it should be `/image-headers`?
Alternatively, simpler to converge both endpoints and have a parameter indicating if it's headers only
cc @carbonin @avishayt

Reply:
`/image.iso` doesn't download only the headers. `/image.iso` does exactly the same thing as `/image`. They both have both `head:` and `get:` methods supported. `head:` returns just the headers, `get:` returns the file.

The only difference is that `/image.iso` doesn't have the `POST` method, which is not needed because `/image.iso` was only created to provide support for Dell iDRAC

Reply:
Sorry, I didn't notice it's a `head` method and there's also another `get` one.
I think this API is really confusing actually. 
Are we going to deprecate the old API?

Reply:
That's definitely an option. We'll look into it once the whole ISO/iDRAC compatibility mess is resolved. Could you please open a ticket for that?

Reply:
https://issues.redhat.com/browse/MGMT-6552

Reply:
Thank you